src/utils/mk64.py

```diff
@@ -0,0 +1,82 @@
+from src.utils import uploader
+
+
+def compare_records(original, newrecords):
+    racer_ids = get_racer_ids()
+    slugs = get_tracklist('slugs')
+
+    for j, (old_times, new_times) in enumerate(zip(original, newrecords)):
+        for i, (or_re, nr_re) in enumerate(zip(old_times['track-record'].records, new_times['track-record'].records)):
+            if i > 4:
+                break
+            if or_re != nr_re:
+                # Commented these out because not being used atm
+                # track_name = old_times['name']
+                # racer_name = racer_ids[nr_re['character']]
+                slug = slugs[j]
+                new_time = nr_re['time']
+                uploader.send_to_gus(new_time, slug, rtype="3lap")
+                break
+
+        if old_times['track-record'].records[5] != new_times['track-record'].records[5]:
+            # Commented these out because not being used atm
+            # track_name = old_times['name']
+            # racer_name = racer_ids[nr_re['character']]
+            new_time = new_times['track-record'].records[5]['time']
+            uploader.send_to_gus(new_time, slugs[j], rtype='flap')
+
+
+def get_racer_ids():
+    return [
+        'Mario',
+        'Luigi',
+        'Yoshi',
+        'Toad',
+        'DK',
+        'Wario',
+        'Peach',
+        ' Bowser'
+    ]
+
+
+def get_tracklist(listType):
```

Comment:
In addition to my question above about putting top-level values inside function calls, this seems like it's a re-implementation of a dictionary with two keys: `names` and `slugs`.

Reply:
Yep, a dictionary works just fine. Just not sure how to set it as a top level variable that can be exported.

Reply:
See https://github.com/SpeedRunTooked/mk64-watcher/pull/1#discussion_r906713099